### DISCUSSION TITLE
Can't upload/paste screenshot in readme

### DISCUSSION THREAD

**Discussion by namratatolani** [community/community#45721] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Bug

### Body

When I try to paste the screenshot or upload the png in readme file, it says something went wrong.
Can't even paste a screenshot in this box.. not sure whats going on..
Message is "Something went really wrong, and we can't process that file. Try again"

**Comment by marianne-s** [comment]
You need to declare the path to your png. 
First, upload your .png file in your github repository
Then copy the following line in your Readme.md file, change the url part by your png url in github. 

```console
 ![alt text](https://github.com/[username]/[repository_name]/blob/[branch]/YourPictureName.png?raw=true) 
```

I hope it help

**Comment by pallabbarman** [comment]
Upload your image to the assets or images folder. Add your image path to the image tag as shown below.

`<img src="/image/sample.png" alt="alt text" />`

or 

`![alt text](/image/sample.png)`

Also, you can use the image url.

**Reply by namratatolani** [reply]
Is it not possible to just add the screenshot directly anymore? It was working perfectly a week back..

**Reply by shodanx2** [reply]
It also no longer works for me
Very annoying to have to save screenshot to a file and then upload that file
It used to be printscreen->region->paste

**Comment by sidx255** [comment]
> ### Select Topic Area
> Bug
> 
> ### Body
> When I try to paste the screenshot or upload the png in readme file, it says something went wrong. Can't even paste a screenshot in this box.. not sure whats going on.. Message is "Something went really wrong, and we can't process that file. Try again"

Were you pasting an image from an online source or was it from a local copy?

**Reply by namratatolani** [reply]
It was screenshot on my linux system with Flameshot app that gets copied to clipboard and gets pasted into readme files or comment boxes on github usually.

**Reply by nerdCopter** [reply]
i have same issue on one computer (xubuntu), but not another (debian). can only paste clipboard images into comments from my debian machine. both are firefox. also use flameshot on both. have not found problem/solution. :confounded:

**Reply by nerdCopter** [reply]
half-hour research ... 
   setting firefox `about:config` `dom.event.clipboardevents.enabled` = `true` solves copy/paste for me.

**Comment by ponomarenkow** [comment]
I have the same problem, although I don't get any message, it's just "uploading your files" and then nothing happens. It worked just a few days ago. 
But it only happens in README files, so a workaround I found is paste/drop the image here in "suggest an answer", then copy the generated tag and paste it into the README file.

**Reply by ponomarenkow** [reply]
Update: problem resolved on its own after several hours.

**Reply by panterasbook29** [reply]
I have had this issue for 2 days already, but it doesn't work in any files, I can't drag and drop or upload images, did it really fix itself?

**Reply by ponomarenkow** [reply]
It definitely worked yesterday, I added a few screenshots directly to the README I was making. But now I checked and it's broken again...

**Reply by snoowfall** [reply]
yep seems like a new issue

**Reply by panterasbook29** [reply]
It has been solved for me, just checked

**Comment by AnkitNayak-eth** [comment]
I had the same issue where **paste/drag in the README editor showed “Uploading…” but nothing appeared in the commit**.

**Temporary workaround:**

1. Go to **Issues → New Issue**
2. Paste or drag the image there
3. GitHub generates a link like
   `https://github.com/user-attachments/assets/...`
4. **Cancel the issue**
5. Copy that link and use it in your README

The image renders normally even if you don’t post the issue.

**Reply by marctison75** [reply]
@AnkitNayak-eth Thanks, I can confirm this workaround works for me too.

One question: if the image is uploaded through a draft/canceled issue and then reused in the README, is that asset guaranteed to remain available in the future, or could it be removed at some point? I’d rather avoid using it if there’s a risk it disappears later.

**Reply by marctison75** [reply]
This is only a temporary workaround. The generated image URL is temporary and will be deleted after a few days, so it is not reliable for a README.

**Comment by marctison75** [comment]
@github Any update on a fix for this?
